torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -688,6 +688,13 @@ void hashValue(Stack* stack) {
   push(stack, int64_t(hash));
 }
 
+// When a number is exactly halfway between two integers, use this trick to
```

Comment:
Nit: explain that this is to match python bahavior

Reply:
done